app/assets/javascripts/application.js

```diff
@@ -10,8 +10,9 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
-//= require rails-ujs
+//= require jquery3
+//= require popper
+//= require bootstrap-sprockets
 //= require activestorage
 //= require turbolinks
-//= require_tree .
-//= require bootstrap-sprockets
\ No newline at end of file
+//= require_tree .
```

Comment:
//= require_tree . **always** has to come last in application.js

Reply:
good to know

Reply:
weird. `//` feels like a comment